buildpack.toml

```
@@ -41,6 +41,12 @@ api = "0.7"
     description = "the application context path"
     name = "BP_TOMCAT_CONTEXT_PATH"
 
+  [[metadata.configurations]]
+    build = true
+    default = "false"
+    description = "Disable the EnvironmentPropertySource for reading environment variables into tomcat config files"
```

```suggestion
    description = "Disable Tomcat's EnvironmentPropertySource"
```

I've been trying to keep the descriptions here as short as possible. Do you think this is sufficient?